Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -588,24 +588,21 @@ def mirror_investigation():
     demisto.results(f'Investigation mirrored successfully, channel: {conversation_name}')
 
 
-def long_running_loop():
+async def long_running_loop():
     while True:
         error = ''
         try:
-            check_for_mirrors()
-            check_for_unanswered_questions()
+            integration_context = check_for_mirrors()
+            check_for_unanswered_questions(integration_context)
```

Comment:
Reducing calls to `demisto.getIntegrationContext()` and passing them between the two. Every time a `demisto` object is called, we lock and unlock the resource so to reduce the need, I'm limiting the number of calls.